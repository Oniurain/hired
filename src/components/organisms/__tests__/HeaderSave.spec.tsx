import * as React from 'react'
import { shallow } from 'enzyme'

import { HeaderSave } from '../'

describe('HeaderSave', () => {
  it('should render', () => {
    const subject = shallow(
      <HeaderSave
        title="New Project"
        cancelHandler={ () => ({}) }
        saveHandler={ () => ({}) }
      />
    )

    expect(subject).toMatchSnapshot()
  })
})
