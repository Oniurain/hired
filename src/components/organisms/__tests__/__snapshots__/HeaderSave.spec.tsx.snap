// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HeaderSave should render 1`] = `
<styled.header>
  <Styled(Component)
    clickHandler={[Function]}
    text="Cancel"
  />
  <styled.h1>
    New Project
  </styled.h1>
  <Styled(Component)
    clickHandler={[Function]}
    text="Save"
  />
</styled.header>
`;
